CONFIGURATION.md

```diff
@@ -16,7 +16,15 @@ Generic placeholders are defined as follows:
 
 The other placeholders are specified separately.
```

Comment:
unrelated to this change but let's link to `example.yml` in this file so users can find it easily :)

```suggestion
The other placeholders are specified separately.

See [example.yml](example.yml) for configuration examples.
```